tools/infrasys/src/error.rs

```diff
@@ -14,7 +14,9 @@ pub enum Error {
     CreateStack {
         stack_name: String,
         region: String,
-        source: rusoto_core::RusotoError<rusoto_cloudformation::CreateStackError>,
+        source: aws_sdk_cloudformation::types::SdkError<
```

Comment:
Nit: I might import the various `*::types` crates  so we don't have to have the fully qualified imports in every single error variant